problems/numberOfDigits.js

```diff
@@ -7,8 +7,13 @@
 
 */
 
-function numberOfDigits() {
-
+function numberOfDigits(n) {
+    let count = 0
+    let num = n.toString()
+    for (let i of num) {
+        count += 1
```

Comment:
What is a property that strings have which provides us with the number of characters in the string? You can use this property in place of the for loop.